frontend/src/pages/new_document.tsx

```diff
@@ -48,11 +47,7 @@ export default function NewDocumentPage() {
   };
 
   return (
-    <div
-      className="h-screen p-6 flex items-center justify-center"
-      onDragOver={(e) => e.preventDefault()}
-      onDrop={(e) => e.preventDefault()}
-    >
+    <AppCenter onDragOver={(e) => e.preventDefault()} onDrop={(e) => e.preventDefault()}>
       <Dialog>
```

Comment:
Can I have a back button for the new document dialog too?

Reply:
Maybe a cancel button instead?